lib/theme/colors.js

```diff
@@ -35,8 +35,8 @@ export const colors = {
   noticeAlt: '#FFFBEF',
   onNotice: '#EB7720',
   warning: '#CB532D',
-  warningAlt: '#FFF6F3',
-  onWarning: '#CB532D',
```

Comment:
How come we're changing this from a red to white? Also looks like it's the same color for light and dark mode? Doesn't seem right.